src/ufoLib2/objects/image.py

```diff
@@ -46,15 +46,15 @@ def __bool__(self) -> bool:
     # the fontTools.ufoLib.validators.imageValidator requires that image is a
     # subclass of Mapping...
 
-    _transformation_keys_: tuple[str, str, str, str, str, str] = (
+    _transformation_keys_: ClassVar[tuple[str, str, str, str, str, str]] = (
```

Comment:
Should we move this stuff to outside the class?

Reply:
well, it is used only by the Image class so it is class-specific. I'll keep ClassVar.